aci/resource_aci_fvrsdomatt.go

```diff
@@ -463,6 +477,36 @@ func resourceAciDomainCreate(ctx context.Context, d *schema.ResourceData, m inte
 		}
 		d.Set("vmm_id", vmmSecP.DistinguishedName)
 	}
+	checkDns := make([]string, 0, 1)
+
+	if relationTofvRsVmmVSwitchEnhancedLagPol, ok := d.GetOk("enhanced_lag_policy"); ok {
+		fvAEPgLagPolAtt := models.NewApplicationEPGLagPolicy(fmt.Sprintf(models.RnfvAEPgLagPolAtt), fvRsDomAtt.DistinguishedName)
+
+		err := aciClient.Save(fvAEPgLagPolAtt)
+		if err != nil {
+			return diag.FromErr(err)
+		}
```

Comment:
isn't this the same as the call to CreateRelationfvRsVmmVSwitchEnhancedLagPol just below?

Reply:
In order to create the relation, we need to create fvAEPgLagPolAtt first and then fvRsVmmVSwitchEnhancedLagPol. Hence, the two calls.